.github/workflows/pr_checks.yml

```diff
@@ -4,9 +4,6 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - fidesctl/**
-      - .github/workflows/pr_checks.yaml
```

Comment:
Did you intend to remove this completely? Should we instead just update it to exclude specific paths?

Reply:
yeah I meant to remove it. I'd rather bias to too much testing than too little. we can start exempting stuff if we want but the majority of the time we'd want testing anyway so 🤷